aws/eks/main.tf

```diff
@@ -21,7 +21,7 @@ resource "aws_iam_role_policy_attachment" "ssm" {
 module "eks" {
   source                        = "terraform-aws-modules/eks/aws"
   create_eks                    = var.create_eks
-  version                       = "~> 13"
+  version                       = "~> 13, < 18.0.0"
```

Comment:
What's the reason for `< 18`? May this break other uses of this module?

Reply:
Version 18 was released a few weeks ago and something has changed because it's broken some arguments when I was trying to run TF using an existing state. When running `init`, it gets the last version by default.

Reply:
This module needs to be re-work to use > 18.
https://github.com/terraform-aws-modules/terraform-aws-eks/blob/master/UPGRADE-18.0.md

Reply:
Did you look into what it takes to make this module work with version 18? Otherwise, could you add a comment here explaining the issue, and a ticket in the backlog to get this fixed? Otherwise we'd be stuck with an old version forever.

Reply:
Ugh, that looks like a lot of changes. Thanks for creating a ticket!